lightwood/encoders/numeric/ts_numeric.py

```diff
@@ -37,7 +37,7 @@ def encode(self, data, extra_data=None):
                 except:
                     real = None
             if self.is_target:
-                vector = [0] * 3
+                vector = [0] * 2
                 if group is not None and self.normalizers is not None:
                     try:
```

Comment:
this is probably unnecessary but why do we have no catch statement here? Do we have any weird edge cases? What happens if the try fails?

Reply:
If the try fails we use the standard normalizer, which considers all training data irrespective of the group to which rows belong. Looking at this, I'm fairly certain there should be no weird uncovered edge cases.